apps/guardian-ui/src/components/VerifyGuardians.tsx

```diff
@@ -334,14 +356,33 @@ export const VerifyGuardians: React.FC<Props> = ({ next }) => {
           <Button
             isDisabled={!verifiedConfigs}
             isLoading={isStarting}
-            onClick={verifiedConfigs ? handleNext : undefined}
+            onClick={handleNext}
             leftIcon={<Icon as={ArrowRightIcon} />}
             width={{ base: 'full', sm: 'auto' }}
           >
             {t('common.next')}
           </Button>
           <WaitingForVerification verifiedConfigs={verifiedConfigs} />
         </Stack>
+        <Modal isOpen={isOpen} onClose={() => setIsOpen(false)}>
+          <ModalOverlay />
+          <ModalContent>
```

Comment:
Add some padding and maybe a title? perhaps a bold "Confirm" then the description below.

And perhaps break up the last sentence into its own paragraph. and even bold/underline it to add emphasis

 
<img width="466" alt="image" src="https://github.com/fedimint/ui/assets/43247027/54e5870b-457e-44e4-87a5-401a9f90faf7">

Reply:
done